src/ethereum-providers/icons/WalletConnect.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note that we could've probably optimized this asset :)

Reply:
In my list of to-dos today. :)